readthedocs/api/v2/views/model_views.py

```diff
@@ -212,9 +213,15 @@ class VersionViewSet(DisableListEndpoint, UpdateModelMixin, UserSelectViewSet):
         'project__slug',
     )
 
+    def get_queryset(self):
+        api_key = self.request.build_api_key
+        if api_key:
+            return self.model.objects.filter(project=api_key.project)
```

Comment:
Should this not call `super().get_queryset().filter(project=api_key.project)`, in case there's other logic acting on the queryset?

Reply:
The queryset from the parent class filters by the projects the current user has access to, when using an api key we don't have a user, so we can't mix them together.

Reply:
That would be a useful comment 👍